src/style.css

```
@@ -1 +1,73 @@
 /* Styles go here */
+.body {
+  margin: 0 auto;
+
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 10px 50px;
```

add just horizontal paddings

```suggestion
  padding: 0 50px;
```

---

and add height 60px to header, list, or each link